src/Tribe/Views/V2/Views/Recent_Past_View.php

```diff
@@ -0,0 +1,154 @@
+<?php
+/**
+ * The Past Recent View.
+ *
+ * @since   TBD
+ * @package Tribe\Events\Views\V2\Views
+ */
+
+namespace Tribe\Events\Views\V2\Views;
+
+use Tribe\Events\Views\V2\View;
+use Tribe\Events\Views\V2\Views\Traits\List_Behavior;
+use Tribe__Context;
+
+class Recent_Past_View extends View {
+
+	use List_Behavior;
+	/**
+	 * Slug for this view
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected $slug = 'recent-past';
+
+	/**
+	 * Visibility for this view.
+	 *
+	 * @since TBD
+	 *
+	 * @var bool
+	 */
+	protected static $publicly_visible = true;
+
+	protected $whitelist = [
+		// Standard View Components.
+		'components/loader',
+		'components/json-ld-data',
+		'components/data',
+		'components/before',
+		'components/messages',
+		'components/breadcrumbs',
+		'components/events-bar',
+		'components/breadcrumbs',
+		'components/top-bar/today',
+		'components/top-bar/actions',
+		'components/events-bar/search/keyword',
+		'components/events-bar/search/submit',
+		'components/events-bar/search-button',
+		'components/events-bar/tabs',
+		'components/events-bar/search',
+		'components/events-bar/filters',
+		'components/events-bar/views',
+
+		// List View.
+		'components/events-bar/views/list',
+		'components/events-bar/views/list/item',
+		'list/top-bar',
+		'list/top-bar/nav',
+		'list/top-bar/datepicker',
+		'list',
+
+		// Recent Past Events Views.
+		'recent-past',
+		'recent-past/event',
+		'recent-past/event/date',
+		'recent-past/event/title',
+		'recent-past/event/venue',
+		'recent-past/event/description',
+		'recent-past/event/cost',
+		'recent-past/event/date-tag',
+		'recent-past/event/date/meta',
+	];
+
+	/**
+	 * Indicates Recent Past View supports the date as a query argument appended to its URL, not as part of a "pretty" URL.
+	 *
+	 * @var bool
+	 */
+	protected static $date_in_url = false;
+
+	/**
+	 * {@inheritDoc}
+	 */
+	protected function setup_repository_args( Tribe__Context $context = null ) {
+		$context = null !== $context ? $context : $this->context;
+		$this->repository = tribe_events();
+
+		$date                   = $context->get( 'event_date', 'now' );
+		$args['posts_per_page'] = $context->get( 'events_per_page', 3 );
+		$args['order_by']       = 'event_date';
+		$args['order']          = 'DESC';
+		$args['ends_before']    = $date;
+
+		return $args;
+	}
+
+	/**
+	 * Add Filters for Whitelist and Adding View HTML.
+	 *
+	 * @since TBD
+	 */
+	public function hook_whitelist_and_html_template() {
```

Comment:
I would rename this to `add_view_filters`.
While we only have this view as an additional View, we might very well have more in the future.
Then we'll refactor additional views to extend an `Addtional_View_Abstract` class of sorts, then a more generic name will allow us to group a lot of common logic in that abstract class and leave the Additional Views implementations deal only with the "configuration".

TL;DR: rename to `add_view_filters`.

Reply:
👍